R/estimate_density.R

```diff
@@ -3,7 +3,7 @@
 #' This function is a wrapper over different methods of density estimation. By default, it uses the base R \link{density} with by default uses a different smoothing bandwidth (\code{"SJ"}) from the legacy default implemented the base R \link{density} function (\code{"nrd0"}). However, Deng \& Wickham suggest that \code{method = "KernSmooth"} is the fastest and the most accurate.
 #'
 #' @inheritParams stats::density
-#' @param method Method of density estimation.
+#' @param method Method of density estimation. Can be \code{"kernel"} (default), \code{"logspline"} or \code{"KernSmooth"}.
```

Comment:
can we rename `KernSmooth` here? Since we use `match.arg()`, arguments can be abbreviated, so it would be nice to have arguments starting with different letters. What about simply `smooth`?

Reply:
Added "smooth" in the last commit is that what you meant?